alert/store.go

```diff
@@ -255,9 +260,63 @@ func NewDB(ctx context.Context, db *sql.DB, logDB alertlog.Store) (*DB, error) {
 			FROM services
 			WHERE id = $1
 		`),
+
+		findPendingNotifications: p(`
+			SELECT
+			COALESCE(ucm.type::text, nc.type::text) AS dest_type,
+			COALESCE(u.name, nc.name) AS dest_name
+			FROM outgoing_messages om
+			LEFT JOIN user_contact_methods ucm 
+			ON om.user_id = ucm.user_id
+			LEFT JOIN notification_channels nc 
+			ON nc.id = om.channel_id
+			LEFT JOIN users u
+			ON u.id = om.user_id
+			WHERE om.last_status='pending'
+			AND (now() - om.created_at) > INTERVAL '1' MINUTE
```

Comment:
I think it would be useful to show this right away since the log is empty until a notification is queued up to send. I was a little confused when testing on the alert details page and nothing was showing up in the log or as a notice at first- I expected to either see that I was being rate-limited, or that my message was being sent

<img width="905" alt="Screen Shot 2021-07-21 at 12 13 00 PM" src="https://user-images.githubusercontent.com/11381794/126546679-2d831bba-c337-481e-a5ac-205a6f97b9d2.png">

Reply:
I'd say 10-15s would be the lowest, 30s would be reasonable.